app/views/_partials/confetti/Constants.ejs

```diff
@@ -0,0 +1,46 @@
+<script type="text/javascript">
+var retina = window.devicePixelRatio,
+
+// Math shorthands
+PI = Math.PI,
+sqrt = Math.sqrt,
+round = Math.round,
+random = Math.random,
+cos = Math.cos,
+sin = Math.sin,
+kappa = (4 * (sqrt(2) - 1))/3
+
+// Local WindowAnimationTiming interface
+rAF = window.requestAnimationFrame,
+cAF = window.cancelAnimationFrame || window.cancelRequestAnimationFrame;
+
+var speed = 50,
+duration = (1.0 / speed),
+confettiRibbonCount = 11,
+ribbonPaperCount = 30,
+ribbonPaperDist = 8.0,
+ribbonPaperThick = 8.0,
+confettiPaperCount = 95,
+balloonCount = 10,
+radius = 50,
+width_factor = 0.0333;
+height_factor = 0.4;
+tie_width_factor = 0.12;
+tie_height_factor = 0.10;
+tie_curve_factor = 0.13;
+graident_factor = 0.3;
```

Comment:
NP, should be `gradient`

Reply:
@sameetandpotatoes what can you clarify this?

Reply:
It's just spelled wrong, not a big deal